selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindTextFeatureTest.java

```diff
@@ -157,19 +204,22 @@ public void checkRecentlyCreatedFiles() throws Exception {
     terminal.waitTerminalTab();
     terminal.selectTerminalTab();
     createFileInTerminal(fileNameCreatedFromTerminal);
-    WaitUtils.sleepQuietly(15);
+    WaitUtils.sleepQuietly(10);
```

Comment:
It's better to user time constants here to be able to increase it from the one place on demand for scalability propose.